src/Log.php

```diff
@@ -271,16 +271,9 @@ public static function history($items_id, $itemtype, $changes, $itemtype_link =
             );
         }
 
-        $old_value = Toolbox::substr($old_value, 0, 180);
-        $new_value = Toolbox::substr($new_value, 0, 180);
-
         // Security to be sure that values do not pass over the max length
-        if (Toolbox::strlen($old_value) > 255) {
-            $old_value = Toolbox::substr($old_value, 0, 250);
-        }
-        if (Toolbox::strlen($new_value) > 255) {
-            $new_value = Toolbox::substr($new_value, 0, 250);
-        }
+        $old_value = Toolbox::substr($old_value, 0, 255);
+        $new_value = Toolbox::substr($new_value, 0, 255);
```

Comment:
IMHO, methods like `Toolbox::substr()` are pretty useless. We could directly use the `mb_substr()` native function.

```suggestion
        $old_value = mb_substr($old_value, 0, 255);
        $new_value = mb_substr($new_value, 0, 255);
```